Tools/sitl_run.sh

```diff
@@ -121,6 +121,14 @@ elif [ "$program" == "gazebo" ] && [ ! -n "$no_sim" ]; then
 		echo "You need to have gazebo simulator installed!"
 		exit 1
 	fi
+elif [ "$program" == "flightgear" ] && [ ! -n "$no_sim" ]; then
+	echo "FG setup"
+    cd "${src_path}/Tools/flightgear_bridge/"
+    "${src_path}/Tools/flightgear_bridge/FG_run.py" ${model}.json    
```

Comment:
What is the behavior if the ${model}.json doesn't exist? - or the model is not supported? It seems like only a plane and TF-G1 is supported. How can we handle people running e.g. `iris` more gracefully rather than doing something random?

Reply:
well, I think that FG_run.py produce some error message (which will lost in output), and bridge will no start due to lack of parameters. PX4 starts but will be waiting for simulator forever.

But it seems that scripts stops after first error (I dont know why):
![rover_error](https://user-images.githubusercontent.com/9033168/78016121-a9b1d700-734a-11ea-8511-9f04587a729d.png)

We can check if backgrounded job FG_run.py is running or not after sleep command. Or have you better idea?
```

Reply:
@slimonslimon It would make sense to run the script for only a limited set of models since currently the list of supported models are quite small.

Reply:
But the list of supported models is set by available .json files. Why should I have list of supported models somewhere and than configuration file for every model?

Or am I missing some configuration of available build targets?

Reply:
when I run gazebo with unsuported model, it produce an error message about missing configuration file and open empty gazebo world, without model....

Reply:
may we can improve CMake skript which generats available target.

Reply:
@slimonslimon Yes that is correct, but the unsupported models are not a make target. All models that are added as a target are supported in gazebo. The problem here is that some predefined make targets will just fail looking like it is just broken.

Reply:
so it mean we should have list of supported models for every supported simulator (supported by make px4_sitl command) 

I will look at this later, because there is other problem because we need build px4 withou LOCKSTEP, and I dont know how to do it in CMake for group of targets (FG targets)

Reply:
both "Lockstep" and "list of models! are solved